ADEN-5054 check isBlocking afer we have info about blocking

dianafa opened the pull request:
https://wikia-inc.atlassian.net/browse/ADEN-5054

dianafa committed:
ADEN-5054

check for presence of gptCreativeSize and gptSlotParams

dianafa committed:
Merge branch 'dev' into ADEN-5054

dianafa committed:
ADEN-5054

call is blocking afer we have info about blocking

nandy-andy commented on extensions/wikia/AdEngine/js/AdInfoTracker.js:
```diff
@@ -30,19 +30,23 @@ define('ext.wikia.adEngine.adInfoTracker',  [
 		};
 
 	function shouldHandleSlot(slot) {
```
What do you think of making it public and unit testing it?

dianafa committed:
ADEN-5054

create ext.wikia.adEngine.adInfoTrackerHelper

dianafa committed:
ADEN-5054

add tests for AdInfoTrackerHelper - shouldHandleSlot

dianafa committed:
ADEN-5054

add tests for AdInfoTrackerHelper - prepareData

dianafa committed:
ADEN-5054

rename getTopLeaderboardSlotWithContainer to getTopLeaderboardSlotWithPageParams
